api/mypaginations.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.response import Response
+
+class MoviesPagination(LimitOffsetPagination):
+  default_limit = 100
+  max_limit = 20
+  DEV_MODE = False # Change to true when running standalone docker container for winearth-api
+  host = 'http://127.0.0.1' if DEV_MODE else 'https://winearth.sdsc.edu'
+
+  def get_paginated_response(self, data):
```

Comment:
My mistake, I didn't implement it correctly in the body of the code. It's supposed to serve as an easy way for me to switch between development testing and what is supposed to be on production. When in development with the docker compose file, the dev site is on 127.0.0.1 and so in order to test if the links work, I have to set the host manually. When in production however, it will use the hostname of the production site `https://winearth.sdsc.edu`. I'm going to fix this so it works as intended. If you don't like my approach to this and would instead rather have me swap it out manually during development i.e. not having the `DEV_MODE` and `host` variables and changing the hostname string directly from `https://winearth.sdsc.edu` to `http://127.0.0.1`, then I can do that as well.